lib/python/treadmill_aws/cli/admin/aws/instance.py

```diff
@@ -137,23 +136,42 @@ def create(
         if not key:
             key = metadata.instance_keys()[0]
 
-        hostnames = hostmanager.create_host(
-            ipa_client=ipa_client,
-            ec2_conn=ec2_conn,
-            image_id=image_id,
-            count=count,
-            disk=disk_size,
-            domain=ipa_domain,
-            key=key,
-            secgroup_ids=secgroup_id,
-            instance_type=size,
-            subnet_id=subnet_id,
-            role=role,
-            instance_vars=instance_vars,
-            instance_profile=instance_profile
+        subnet_ids = set(
+            [aws_cli.admin.subnet_id(ec2_conn, network) for network in subnet]
         )
-        for hostname in hostnames:
-            click.echo(hostname)
+
+        total_subnets = list(
+            [ec2client.list_subnets(ec2_conn, subnet_id.split()) for subnet_id
+             in subnet_ids]
+        )
+
+        value = 1
```

Comment:
not needed. aside from that, it is not a good variable name. subnet_idx is better (yet this is not needed at all).